_examples/openapi.gen.yaml

```diff
@@ -537,6 +537,9 @@ paths:
   /rpc/ExampleService/GetUserV2:
     post:
       summary: GetUserV2
+      security:
+          - ApiKeyAuth: []
+          - ServiceAuth: []
```

Comment:
```
webrpc-gen -target openapi -securitySchemeNameAnnotation=@auth -securitySchemes='{"ApiKeyAuth":....}'`
```
what do you think?

Reply:
feels to verbose :): so if you can simplify this, it'd be great

Reply:
I don't know how to do it actually. 

If I pass `json` object or array of json objects I would need to parse it to a custom `struct` and then render slice of these structs to `yaml` format..

If `-securitySchemes` is a custom template option variable then I would have to do a specific check in `webrpc` 


```
if name == "securitySchemes"  {
  json.Marshal(`bytes`, &customStruct)
}
```

https://github.com/webrpc/webrpc/blob/f0558f21541cc3ae1654014c82d55a6ed4a07bfc/cmd/webrpc-gen/main.go#L141

Reply:
Can we pass it as a raw (one-liner) string and `printf "%s" .` it from the template?

IDK if it's possible but something like `[{""},{""}]`

Javascript is subset of YAML, isn't it?